apps/seg/itkimage2segimage.xml

```diff
@@ -64,6 +64,15 @@
       <description>Skip empty slices while encoding segmentation image. By default, empty slices will not be encoded, resulting in a smaller output file size.</description>
     </boolean>
 
+    <boolean>
```

Comment:
I used this approach for two reasons:

1. Don't break backward compatibility
2. Make sure that conversion succeeds also in cases where label IDs cannot be sorted (since then the call will fail). I assume the user does often not know beforehand, whether the labelmaps will allow sorting. So the idea is that you only turn on this functionality if the user knows the prerequisites for this are fulfilled.

If 1. does not matter, one could also enable it by default and fall back to the old behavior if re-sorting does not work. I am not sure whether this can be implemented well, since the "rollback" after half of the re-resorting is probably difficult. And finding this out before actually starting the sorting work costs extra time.

Reply:
I agree! I made this comment in the beginning of my review, and if I revisited it after the review was done, I would most definitely revert it!